src/test/java/com/meilisearch/integration/TasksTest.java

```diff
@@ -202,6 +205,38 @@ public void testClientGetTasksAllQueryParameters() throws Exception {
         assertThat(result.getNext(), is(notNullValue()));
         assertThat(result.getResults().length, is(notNullValue()));
     }
+    /** Test to check if task list is reversed when enabled */
```

Comment:
No need to explain what you're testing, this is usually a smell that your test is too complex. Which is not the case :)